dandi/publish/views/draft_version.py

```diff
@@ -83,3 +88,36 @@ def draft_publish_view(request, dandiset__pk):
     dandiset.draft_version.lock(request.user)
     publish_version.delay(dandiset.id, request.user.id)
     return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+@swagger_auto_schema(
+    method='POST',
+    request_body=UserSerializer(many=True),
+)
+@api_view(['POST'])
+@permission_required_or_403('owner', (DraftVersion, 'dandiset__pk', 'dandiset__pk'))
+@permission_classes([IsAuthenticatedOrReadOnly])
+def draft_owners_view(request, dandiset__pk):
+    draft_version = get_object_or_404(Dandiset, pk=dandiset__pk).draft_version
+
+    new_owners_serializer = UserSerializer(data=request.data, many=True)
+    if not new_owners_serializer.is_valid():
+        return Response(new_owners_serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+    new_owners = [
+        get_object_or_404(User, username=owner['username'])
+        for owner in new_owners_serializer.validated_data
+    ]
+    if len(new_owners) < 1:
+        return Response('Cannot remove all draft owners', status=status.HTTP_400_BAD_REQUEST)
+    old_owners = get_users_with_perms(draft_version, only_with_perms_in=['owner'])
+
+    # Remove old owners
+    for old_owner in old_owners:
+        if old_owner not in new_owners:
+            remove_perm('owner', old_owner, draft_version)
+    # Add new owners
+    for new_owner in new_owners:
+        if new_owner not in old_owners:
+            assign_perm('owner', new_owner, draft_version)
+
+    return Response('', status=status.HTTP_204_NO_CONTENT)
```

Comment:
```suggestion
    return Response(status=status.HTTP_204_NO_CONTENT)
```
The `data` parameter is optional too.